pageserver/src/tenant.rs

```diff
@@ -2084,7 +2106,8 @@ impl Tenant {
         );
 
         // Protect against concurrent attempts to use this TimelineId
-        let create_guard = self.create_timeline_create_guard(new_timeline_id)?;
+        let allow_offloaded = false;
```

Comment:
I don't see `allow_offloaded` being changed across the patch, probably just put `false` directly at the parameter, or you feel it's clearer to make a separate variable for that? (no strong opinion)

Reply:
it's set in `unoffload_timeline`